tests/core/gas-strategies/test_time_based_gas_price_strategy.py

```diff
@@ -19,8 +19,7 @@ def _get_block_by_something(method, params):
     block_identifier = params[0]
     if (
         block_identifier == 'latest' or
-        block_identifier == '0x0000000000000000000000000000000000000000000000000000000000000005' or
```

Comment:
curious about the difference/requirement of prefixed zeros here 🤔

Reply:
This particular block hash just wasn't used in the tests. I could leave it in if you want, I don't have a strong opinion either way. The other block identifier hashes are used with the `getBlockByHash` method below. But I did have to change `5` to `0x5` to account for data normalization that wasn't happening before, but now is.

Reply:
👍 If unused, def leave it out.